modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -11,234 +11,41 @@ NOTE: Couchbase Lite 3.0 introduces some breaking changes. +
 If you're upgrading from 2.x, refer to the appropriate upgrade page -- see: <<lbl-upgrade>>. +
 You should be able to upgrade from 3.0.x to 3.1.x without manual intervention.
 
-== Release 3.2.3 (April 2025)
+== Release 3.3.0 (August 2025)
 
 === New Features
 
-==== Swift API Enhancements
+=== Multipeer Replicator
 
-Couchbase Lite 3.2.3 introduces Reactive APIs in Swift.
-The Reactive APIs enable you map Couchbase Lite documents directly to Swift models, and then bind a model to a view representing the user interface.
-The user interface is automatically updated when the underlying data changes.
-The new Reactive APIs provide the following benefits:
+Couchbase Lite 3.3.0 introduces the new Multipeer Replicator API for peer-to-peer synchronization in both Swift and Kotlin platforms.
```

Comment:
Multipeer Replicator API for peer-to-peer synchronization
-->
Multipeer Replicator API for bidirectional peer-to-peer synchronization

...as CBL already had p2p, sans bidirectionality.